include/ibamr/INSCollocatedHierarchyIntegrator.h

```diff
@@ -183,12 +183,14 @@ class INSCollocatedHierarchyIntegrator : public INSHierarchyIntegrator
                                        bool skip_synchronize_new_state_data,
                                        int num_cycles = 1) override;
 
-    /*!
-     * Regrid the patch hierarchy.
+protected:
+    /*
+     * Since (unlike the staggered case) there is no known divergence-free
```

Comment:
One more change to the comment here to try to clarify:
```suggestion
     * Since (unlike the staggered case) there is no local divergence-preserving
```
There are non-local interpolation schemes (e.g. involving solving Poisson problems).